db/migrate/20200924121000_migrate_old_degree_grade_values_to_hesa_descriptions.rb

```diff
@@ -0,0 +1,15 @@
+class MigrateOldDegreeGradeValuesToHesaDescriptions < ActiveRecord::Migration[6.0]
+  def change
```

Comment:
As this is non-reversible (and doesn't need to be) would it be best to make this an `up` and have an empty `down` method?

Reply:
Doesn't trigger an exception on rollback which I think is probably fine?